services/fxart.ts

```diff
@@ -145,6 +147,26 @@ export const allocateClaim = async (body, id) => {
   }
 }
 
+export const updateMetadata = async ({ chain, collection, nft, metadata }) => {
+  try {
+    const response = await api<DoResult>('/metadata/v1/update', {
+      method: 'get',
+      query: {
+        chain,
+        collection,
+        nft,
+        metadata,
+      },
+    })
+
+    return response
```

Comment:
1. It should be `POST` method instead of `GET` to update info.
2. Is it safe to provide this API for everyone to update the metadata without any limit?

Reply:
updated, this is the PR on fxart https://github.com/kodadot/private-workers/pull/105

Reply:
> updated, this is the PR on fxart https://github.com/kodadot/private-workers/pull/105

thanks!

I saw you talk about this security topic in the issue description.   https://github.com/kodadot/private-workers/issues/99
> Someone can override another metadata

It's not related to this PR then.

Reply:
> I saw you talk about this security topic in the issue description

yes, since you mentioned about "safe" I added [409 status](https://github.com/kodadot/private-workers/pull/105/commits/00d2456e5c872d15ddec58614498da0ca2d86d59). I just remembered that I have some concern about that. then I put it on that issue description just in case I forgot again 😅

thank you anyway